### DISCUSSION TITLE
Setup for NuScenes submission file generation

### DISCUSSION THREAD

**Discussion by MicheleRep** [open-mmlab/mmdetection3d#2267] [Q&A]
Hi, I am trying to generate a json file of labels for the NuScenes Object Detection Challenge submission, because I would like to work on this labels format.
I have done all the steps listed [here](https://mmdetection3d.readthedocs.io/en/stable/getting_started.html#installation) in  but when it comes to the NuScenes Dataset preparation command as specified [here](https://mmdetection3d.readthedocs.io/en/stable/datasets/nuscenes_det.html) it returns the error:

> AssertionError: MMCV==1.7.1 is used but incompatible. Please install mmcv>=1.3.17, <=1.5.0.

but when I install one of the compatible version like mmcv==1.4.0 it returns:

> AssertionError: MMCV==1.4.0 is used but incompatible. Please install mmcv>=1.5.2, <=1.7.0

but when I install one of the compatible version like mmcv==1.5.2 it returns:

> AssertionError: MMCV==1.5.2 is used but incompatible. mmcv>=1.3.17, <=1.5.0.

and so on.

Anyone can help me or someone that already have done this could tell me how to do?
Thank you

**Comment by JingweiZhang12** [comment]
Please provide all the error logs so we can see which library has a conflict with the MMCV. @MicheleRep
